src/components/Confirm/index.js

```diff
@@ -163,8 +163,9 @@ class Confirm extends Component  {
     const duration = Math.round(performance.now() - this.startTime)
     sendEvent('Completed upload', {duration, method: this.props.method})
     this.setState({onfidoId: apiResponse.id})
+    const shouldWarnGlareDetection = this.props.documentType !== 'poa'
     const warnings = apiResponse.sdk_warnings
-    if (warnings && !warnings.detect_glare.valid) {
+    if (warnings && !warnings.detect_glare.valid && shouldWarnGlareDetection) {
```

Comment:
this is the warning that comes from the api, right? If so we should disabled the validation when we make the call not after receiving the result

Reply:
@rfreitas, are you suggesting we inform the API it should not detect glare? Is there a way to do so?

Reply:
yes, here: https://github.com/onfido/onfido-sdk-ui/blob/development/src/components/utils/onfidoApi.js#L16

At the moment some business logic is in that util file, which is not very good. if we can surface it a little bit, it would be better

Reply:
but essentially there is a body parameter `sdk_validations: {'detect_document': 'error'|'warn, 'detect_glare': 'error'|'warn'}`

Reply:
Great, I completely missed it. Thanks @rfreitas